include/deal.II/lac/affine_constraints.templates.h

```diff
@@ -448,7 +448,7 @@ namespace internal
     std::vector<types::global_dof_index> constrained_indices_temp;
     for (const auto &line : constraints_in.get_lines())
       constrained_indices_temp.push_back(line.index);
```

Comment:
I agree with the change below. While here, could you please also add
```suggestion
    std::vector<types::global_dof_index> constrained_indices_temp;
    constrained_indices_temp.reserve(constraints_in.n_constraints());
    for (const auto &line : constraints_in.get_lines())
      constrained_indices_temp.push_back(line.index);
```